src/addrman.h

```diff
@@ -261,7 +255,7 @@ friend class CAddrManTest;
 #endif
 
     //! Select several addresses at once.
-    void GetAddr_(std::vector<CAddress> &vAddr) EXCLUSIVE_LOCKS_REQUIRED(cs);
+    void GetAddr_(std::vector<CAddress> &vAddr, unsigned int max_addresses, unsigned int max_pct) EXCLUSIVE_LOCKS_REQUIRED(cs);
```

Comment:
`size_t` might be more appropriate for `max_addresses`

Reply:
ok, changed all of these to size_t